super32emu/super32emu/ui/emulator_widget.py

```diff
@@ -86,7 +88,7 @@ def __create_register_group(self):
                 self.register_layout.addWidget(r, x, y, Qt.AlignRight)
                 current_register += 1
 
-        self.z_register = RegisterWidget("Z")
+        self.z_register = RegisterWidget("Z", mask="H")
```

Comment:
```suggestion
        self.z_register = RegisterWidget("Z", mask="B")
```

Z flag is a one bit value. This mask should only accept 1 or 0 as value.